app/Models/Wishlist.php

```diff
@@ -16,4 +16,9 @@ public function toString(): string
     {
         return "Welcome to your wishlist! Hopefully you get $this->title";
     }
+
```

Comment:
this class is our Pivot table between users and games, so it needs to extend the Pivot class as mentioned here: https://laravel.com/docs/9.x/eloquent-relationships#defining-custom-intermediate-table-models

Also we need to add a games property to the Users table that uses this Pivot class, as mentioned in the above link.

Reply:
Okay preeeeetty sure this is resolved!